bin/taxref_reformat_gg2.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Handles the Greengenes2 DADA2 database.
+
+# Write the assignTaxonomy() fasta file: assignTaxonomy.fna
+zcat *.gz | sed '/^>/s/>[^ ]\+ \([^[]\+\) \[.*/>\1/' | sed '/^>/s/ \[.*//' | sed 's/[a-z]__//g' | gzip -c > assignTaxonomy.fna.gz
```

Comment:
These are pretty gnarly! Could be worth spelling out in a comment what they are doing, step-by-step?